File: backend/src/main/java/org/jdk/project/dto/config/AreaDto.java
package org.jdk.project.dto.config;

import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class AreaDto {
  private Long id;
  private String areaName;
  private String roomCode;
  private String deptName;
  private String coordinate;
  private Integer maxCapacity;
  private String isGuide;
  private String isShow;
  private Integer enable;
  private String remark;
  private List<AreaDetailDto> configAreaDetailList;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/AreaDto.kt
package org.jdk.project.dto.config

data class AreaDto(
    var id: Long? = null,
    var siteId: Long? = null,
    var siteCode: String? = null,
    var siteName: String? = null,
    var areaName: String? = null,
    var coordinate: String? = null,
    var maxCapacity: Int? = null,
    var isShow: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
